src/main/java/com/iota/iri/cache/InvalidCacheConfigurationException.java

```diff
@@ -0,0 +1,8 @@
+package com.iota.iri.cache;
+
+public class InvalidCacheConfigurationException extends RuntimeException {
+
+    public InvalidCacheConfigurationException(String message) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Public method and constructor comments are required](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4567264)